section2/exercises/if_statements.rb

```diff
@@ -61,5 +64,11 @@
 # Experiment with manipulating the value held within both variables
 # to make sure all above conditions output what you expect.
 
-cups_of_flour = 1
-has_sauce = true
+cups_of_flour = 3
+has_sauce = true 
+
+if (cups_of_flour >= 2) && (has_sauce == true)
```

Comment:
Great. Parenthesis are technically optional here (and as Rubyists we *generally* to omit unnecessary symbols)